app/views/admin/communities/maps.haml

```diff
@@ -0,0 +1,24 @@
+- content_for :title_header do
+  %h1
+    = t("layouts.admin.admin")
+    = "-"
+    = t("admin.communities.maps.maps")
+
+= render :partial => "admin/left_hand_navigation", :locals => { :links => admin_links_for(@current_community) }
+
+.left-navi-section
+  %h2= t(".maps", :community_name => @community.name(I18n.locale))
+  = form_for @community, :url => maps_admin_community_path(@community), :method => :put do |form|
+    .row
+      %h3= t(".google_maps_key")
```

Comment:
We have agreed to use full paths for translations keys `admin.communities.maps.google_maps_key`. The problem with the shorter version is that they are tightly coupled to the filename and directory, so renaming the file or moving it breaks the translations.